### ISSUE TITLE
retrieve loss function values from checkpoints 

### ISSUE THREAD

**Issue by h-ann** [thtrieu/darkflow#918]
Hi, everyone! 

I've trained the model. Is it possible now to retrieve the loss function values from the model checkpoints?
For example, in order to plot the dependency of loss value from step #. 

Thanks.

**Comment by sandeeprepakula** [comment]
It is not possible to retrieve loss values from checkpoints. However if you have used --summary logdirectoryForTensorboard option while training the model then you can see the loss values against each step in tensorboard using the folder containing logs for tensorboard loss visualisation.

**Comment by h-ann** [comment]
Thanks
